### PR TITLE
Bus mem reviewer changes

### PR DESCRIPTION
**Pull request by valasensio**
#### What's this PR do?
Implements reviewers' requested changes

#### Why are we doing this? How does it help us?
Polished the message, copyedit improvements

#### How should this be manually tested?
No tests per se; the original branch merge is where the action is

#### How should this change be communicated to end users?
Already has been communicated

#### Are there any smells or added technical debt to note?
There is some duplication of front-end code between the the different `entry` points in the Donations app. 

#### What are the relevant tickets?
There are a number of related Basecamp components, but this is the main group:
[Business Membership Group](https://3.basecamp.com/3098728/projects/9070027)

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***

* [ ] Added automated tests? *(not yet )*
* [x] Tested manually on mobile? *( )*
* [ ] Checked for performance implications? *(n/a)*
* [x] Checked for security implications? *( )*
* [x] Updated the documentation/wiki? *( )*

#### TODOs / next steps:
There will probably be some tweaks required after initial launch

### CODE REVIEW

File: templates/includes/benefits.html [hunk 1]
```diff
@@ -34,10 +34,20 @@ <h4 class="grid_separator--s" ga-event-category="donations" ga-event-action="ben
 
 <h4 class="grid_separator--s">Giving circles</h4>
 <div class="prose grid_separator--l">
-  <p>Circle members provide sustainable support with three-year commitments to fund our newsroom. Members receive all of the Involved benefits plus access to exclusive events and behind-the-scenes information.</p>
+  <p>Circle members provide sustainable support with three-year commitments to fund our newsroom. Members receive all of the Involved benefits plus access to exclusive events and behind-the-scenes information. <a href="https://stripe-testing.texastribune.org/circleform"> Read more about the Circle Membership </a></p>
```
Comment (AndrewGibson27) [inline templates/includes/benefits.html]: That link is going to the test server

Comment (valasensio) [reply templates/includes/benefits.html]: Crikey. That was in my notes to self to change before production deploy. Great catch!!!

File: templates/includes/benefits.html [hunk 2]
```diff
@@ -34,10 +34,20 @@ <h4 class="grid_separator--s" ga-event-category="donations" ga-event-action="ben
...
 </div>
 <ul class="list--bulleted grid_separator--l">
   <li>Editor’s Circle - $1,000/year for three years</li>
   <li>Leadership Circle - $2,500/year for three years</li>
   <li>Chairman’s Circle - $5,000/year for three years</li>
 </ul>
+
+<h4 class="grid_separator--s">Business membership</h4>
+<div class="prose grid_separator--l">
+  <p>Make a contribution in your organization's name and join a roster of community-minded businesses supporting our public interest newsroom.  Members are featured on a support page with link to business site, mentioned on The Brief, and receive an official Texas Tribune Business Member decal and card. <a href="https://stripe-testing.texastribune.org/business"> Read more about the Business Membership </a></p>
```
Comment (x110dc) [inline templates/includes/benefits.html]: `stripe-testing.texastribune.org` here too.

**Comment by x110dc** [comment]
Just those two minor fixes. Otherwise looks great!

**Comment by valasensio** [comment]
I think that's everything.

**Pull request merged**